content/tutorials/history.textile

```diff
@@ -193,14 +215,22 @@ blang[python].
   "See this step in Github":https://github.com/ably/tutorials/commit/6a350c2
 
 blang[swift].
+  If you never had a chance to build a basic user interface in Xcode, please refer to "Apple developer guide":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/Lesson2.html on how to build a simple UI and also learn more about "adding @@IBOutlets@ and @@IBActions@":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/Lesson3.html#//apple_ref/doc/uid/TP40015214-CH22-SW1.
+  To be able to subscribe to a channel and send some initial messages add a button that will do that. Go to your @ExampleViewController@ and add an @UIButton@ from @Object library@. Name the action "publishAction".
```

Comment:
Change to "To be able to subscribe to a channel and send some initial messages, we will add a listener for new messages and add a button that publishes a message."